Migrate internal macOS Sequoia test baselines/expectations to OpenSource.

bls1999 opened the pull request:
#### be12f26c80c579ebac9e2b9477e7b626b8872dde
<pre>
Migrate internal macOS Sequoia test baselines/expectations to OpenSource.
<a href="https://bugs.webkit.org/show_bug.cgi?id=276270">https://bugs.webkit.org/show_bug.cgi?id=276270</a>
<a href="https://rdar.apple.com/131192311">rdar://131192311</a>

Reviewed by Ryan Haddad.

Migrating macOS Sequoia layout test baselines and expectations from Internal -&gt; OpenSource.

* LayoutTests/platform/mac-sonoma-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Copied from LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt.
* LayoutTests/platform/mac-sonoma-wk1/platform/mac/fast/AppleScript/date-expected.txt: Added.
* LayoutTests/platform/mac-sonoma-wk1/svg/text/caret-in-svg-text-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/css2.1/20110323/c543-txt-decor-000-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counter-03-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counter-03-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counter-04-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counter-04-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counters-03-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counters-03-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counters-04-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counters-04-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css3/unicode-bidi-isolate-basic-expected.txt: Copied from LayoutTests/platform/mac/css3/unicode-bidi-isolate-basic-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-2-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-2-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-2-left-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-2-left-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-right-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-right-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-002-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-002-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-009a-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-009a-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-012-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-012-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-bidi-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-bidi-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-strict-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-strict-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/dom/34176-expected.txt: Copied from LayoutTests/platform/mac/fast/dom/34176-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/dom/52776-expected.txt: Copied from LayoutTests/platform/mac/fast/dom/52776-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-long-image-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-long-image-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-repeat-x-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-x-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-repeat-y-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-y-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-letter-spacing-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-letter-spacing-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-mixed-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-rl-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-rl-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/bidi-embedding-pop-and-push-same-expected.txt: Copied from LayoutTests/platform/mac/fast/text/bidi-embedding-pop-and-push-same-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/emphasis-expected.txt: Copied from LayoutTests/platform/mac/fast/text/emphasis-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-AN-after-L-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-AN-after-L-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-001-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-001-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-002-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-002-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-003-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-003-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-mirror-he-ar-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-mirror-he-ar-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-neutral-run-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-neutral-run-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/hebrew-vowels-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/hebrew-vowels-expected.txt.
* LayoutTests/platform/mac-sonoma/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Copied from LayoutTests/platform/mac/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt.
* LayoutTests/platform/mac-sonoma/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt: Copied from LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt.
* LayoutTests/platform/mac-sonoma/platform/mac/fast/text/international/bidi-fallback-font-weight-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/platform/mac/fast/text/line-break-locale-expected.txt: Copied from LayoutTests/platform/mac/platform/mac/fast/text/line-break-locale-expected.txt.
* LayoutTests/platform/mac-sonoma/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt: Copied from LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt.
* LayoutTests/platform/mac-sonoma/svg/custom/glyph-selection-bidi-mirror-expected.txt: Copied from LayoutTests/platform/mac/svg/custom/glyph-selection-bidi-mirror-expected.txt.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt:
* LayoutTests/platform/mac-wk1/platform/mac/fast/AppleScript/date-expected.txt: Added.
* LayoutTests/platform/mac-wk1/svg/text/caret-in-svg-text-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/mac/css2.1/20110323/c543-txt-decor-000-expected.txt: Added.
* LayoutTests/platform/mac/css2.1/t1202-counter-03-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counter-04-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counters-03-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counters-04-b-expected.txt:
* LayoutTests/platform/mac/css3/unicode-bidi-isolate-basic-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-2-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-2-left-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-right-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-002-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-009a-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-012-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-bidi-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-strict-expected.txt:
* LayoutTests/platform/mac/fast/dom/34176-expected.txt:
* LayoutTests/platform/mac/fast/dom/52776-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-long-image-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-x-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-y-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-letter-spacing-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-mixed-expected.txt: Added.
* LayoutTests/platform/mac/fast/ruby/bopomofo-rl-expected.txt:
* LayoutTests/platform/mac/fast/text/bidi-embedding-pop-and-push-same-expected.txt:
* LayoutTests/platform/mac/fast/text/emphasis-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-AN-after-L-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-001-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-002-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-003-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-mirror-he-ar-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-neutral-run-expected.txt:
* LayoutTests/platform/mac/fast/text/international/hebrew-vowels-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt:
* LayoutTests/platform/mac/platform/mac/fast/text/international/bidi-fallback-font-weight-expected.txt: Added.
* LayoutTests/platform/mac/platform/mac/fast/text/line-break-locale-expected.txt:
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt:
* LayoutTests/platform/mac/svg/custom/glyph-selection-bidi-mirror-expected.txt:

Canonical link: <a href="https://commits.webkit.org/280704@main">https://commits.webkit.org/280704@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7c22d62814a7816580c9c64e115e74f1eff71b36

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/57375 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/36703 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/9850 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/60997 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/7820 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/44327 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/8008 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/46464 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/5534 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/59405 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/34442 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/49552 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/27328 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/31224 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/6864 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/6823 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/53235 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/7135 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/62676 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/1288 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/7228 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/53725 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/1294 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/49587 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/53812 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12679 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/1100 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/32532 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/33617 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/34702 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/33363 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad reviewed (approved):
rs=me assuming EWS is happy.

bls1999 committed:
Migrate internal macOS Sequoia test baselines/expectations to OpenSource.
https://bugs.webkit.org/show_bug.cgi?id=276270
rdar://131192311

Reviewed by Ryan Haddad.

Migrating macOS Sequoia layout test baselines and expectations from Internal -> OpenSource.

* LayoutTests/platform/mac-sonoma-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Copied from LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt.
* LayoutTests/platform/mac-sonoma-wk1/platform/mac/fast/AppleScript/date-expected.txt: Added.
* LayoutTests/platform/mac-sonoma-wk1/svg/text/caret-in-svg-text-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/css2.1/20110323/c543-txt-decor-000-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counter-03-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counter-03-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counter-04-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counter-04-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counters-03-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counters-03-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css2.1/t1202-counters-04-b-expected.txt: Copied from LayoutTests/platform/mac/css2.1/t1202-counters-04-b-expected.txt.
* LayoutTests/platform/mac-sonoma/css3/unicode-bidi-isolate-basic-expected.txt: Copied from LayoutTests/platform/mac/css3/unicode-bidi-isolate-basic-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-2-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-2-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-2-left-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-2-left-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-expected.txt.
* LayoutTests/platform/mac-sonoma/editing/selection/caret-rtl-right-expected.txt: Copied from LayoutTests/platform/mac/editing/selection/caret-rtl-right-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-002-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-002-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-009a-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-009a-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/borders/bidi-012-expected.txt: Copied from LayoutTests/platform/mac/fast/borders/bidi-012-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-bidi-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-bidi-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/css/text-overflow-ellipsis-strict-expected.txt: Copied from LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-strict-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/dom/34176-expected.txt: Copied from LayoutTests/platform/mac/fast/dom/34176-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/dom/52776-expected.txt: Copied from LayoutTests/platform/mac/fast/dom/52776-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-long-image-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-long-image-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-repeat-x-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-x-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/inline/inline-box-background-repeat-y-expected.txt: Copied from LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-y-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-letter-spacing-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-letter-spacing-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-mixed-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/ruby/bopomofo-rl-expected.txt: Copied from LayoutTests/platform/mac/fast/ruby/bopomofo-rl-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/bidi-embedding-pop-and-push-same-expected.txt: Copied from LayoutTests/platform/mac/fast/text/bidi-embedding-pop-and-push-same-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/emphasis-expected.txt: Copied from LayoutTests/platform/mac/fast/text/emphasis-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-AN-after-L-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-AN-after-L-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-001-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-001-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-002-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-002-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-linebreak-003-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-linebreak-003-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-mirror-he-ar-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-mirror-he-ar-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/bidi-neutral-run-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/bidi-neutral-run-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/text/international/hebrew-vowels-expected.txt: Copied from LayoutTests/platform/mac/fast/text/international/hebrew-vowels-expected.txt.
* LayoutTests/platform/mac-sonoma/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Copied from LayoutTests/platform/mac/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt.
* LayoutTests/platform/mac-sonoma/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt: Copied from LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt.
* LayoutTests/platform/mac-sonoma/platform/mac/fast/text/international/bidi-fallback-font-weight-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/platform/mac/fast/text/line-break-locale-expected.txt: Copied from LayoutTests/platform/mac/platform/mac/fast/text/line-break-locale-expected.txt.
* LayoutTests/platform/mac-sonoma/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt: Copied from LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt.
* LayoutTests/platform/mac-sonoma/svg/custom/glyph-selection-bidi-mirror-expected.txt: Copied from LayoutTests/platform/mac/svg/custom/glyph-selection-bidi-mirror-expected.txt.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt:
* LayoutTests/platform/mac-wk1/platform/mac/fast/AppleScript/date-expected.txt: Added.
* LayoutTests/platform/mac-wk1/svg/text/caret-in-svg-text-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/mac/css2.1/20110323/c543-txt-decor-000-expected.txt: Added.
* LayoutTests/platform/mac/css2.1/t1202-counter-03-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counter-04-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counters-03-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counters-04-b-expected.txt:
* LayoutTests/platform/mac/css3/unicode-bidi-isolate-basic-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-2-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-2-left-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-expected.txt:
* LayoutTests/platform/mac/editing/selection/caret-rtl-right-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-002-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-009a-expected.txt:
* LayoutTests/platform/mac/fast/borders/bidi-012-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-bidi-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-expected.txt:
* LayoutTests/platform/mac/fast/css/text-overflow-ellipsis-strict-expected.txt:
* LayoutTests/platform/mac/fast/dom/34176-expected.txt:
* LayoutTests/platform/mac/fast/dom/52776-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-long-image-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-x-expected.txt:
* LayoutTests/platform/mac/fast/inline/inline-box-background-repeat-y-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-letter-spacing-expected.txt:
* LayoutTests/platform/mac/fast/ruby/bopomofo-mixed-expected.txt: Added.
* LayoutTests/platform/mac/fast/ruby/bopomofo-rl-expected.txt:
* LayoutTests/platform/mac/fast/text/bidi-embedding-pop-and-push-same-expected.txt:
* LayoutTests/platform/mac/fast/text/emphasis-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-AN-after-L-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-001-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-002-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-linebreak-003-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-mirror-he-ar-expected.txt:
* LayoutTests/platform/mac/fast/text/international/bidi-neutral-run-expected.txt:
* LayoutTests/platform/mac/fast/text/international/hebrew-vowels-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/text/reftests/tspan-opacity-mixed-direction-expected.txt:
* LayoutTests/platform/mac/platform/mac/fast/text/international/bidi-fallback-font-weight-expected.txt: Added.
* LayoutTests/platform/mac/platform/mac/fast/text/line-break-locale-expected.txt:
* LayoutTests/platform/mac/svg/W3C-SVG-1.1/text-tselect-02-f-expected.txt:
* LayoutTests/platform/mac/svg/custom/glyph-selection-bidi-mirror-expected.txt:

Canonical link: https://commits.webkit.org/280704@main
